apps/classifications/signals.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+import backoff
+import httpx
+from django.conf import settings
+from django.db.models import signals
+from django.dispatch import receiver
+
+from adhocracy4.comments.models import Comment
+
+from .models import AIClassification
+
+client = httpx.Client()
+logger = logging.getLogger(__name__)
+
+
+@receiver(signals.post_save, sender=Comment)
+def get_ai_classification(sender, instance, **kwargs):
+    try:
+        response = call_ai_api(comment=instance)
+        if response.status_code == 200 \
+                and response.json()['classification'] == 'OFFENSE':
+
+            classification = AIClassification(
+                comment=instance,
+                classification='OFFENSIVE')
+            classification.save()
+    except httpx.HTTPError:
+        logger.error('Error connecting to %s', settings.AI_API_URL)
```

Comment:
I am hoping that this way the error will appear in sentry, but not sure if this works?